test/neura/fusion/test.mlir

```diff
@@ -0,0 +1,19 @@
+# RUN: clang++ -S -emit-llvm -o kernel.ll kernel.cpp
+# RUN: mlir-translate --import-llvm kernel.ll -o kernel.mlir
+# RUN: mlir-neura-opt --assign-accelerator \
+# RUN:           --lower-llvm-to-neura \
+# RUN:           --canonicalize-live-in \
+# RUN:           --leverage-predicated-value \
+# RUN:           --fold-constant \
+# RUN:           --transform-ctrl-to-data-flow \
+# RUN:           --fold-constant \
+# RUN:           --fuse-pattern \
+# RUN:           --view-op-graph \
+# RUN:           --insert-data-mov kernel.mlir | FileCheck %s --check-prefix=CHECK-FUSED
```

Comment:
Can you please copy the generated DFG.png in the PR's description (before and after), so we can visualize the fusion?